doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc

```diff
@@ -7,7 +7,7 @@ With {PRODUCT}, you can set up a development environment to develop business app
 
 * *Runtime environment*: Consists of one or more {KIE_SERVER} instances with or without {CENTRAL}. {CENTRAL} has an embedded {CONTROLLER}. If you install {CENTRAL}, use the *Menu* -> *Deploy* -> *Execution servers* page to create and maintain containers. If you want to automate {KIE_SERVER} management without {CENTRAL}, you can use the {HEADLESS_CONTROLLER}.
 
-You can also cluster both development and runtime environments. A clustered development or runtime environment consists of a unified group or "cluster" of two or more servers. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration, while the primary benefit of clustering {PRODUCT} runtime environments is high availability and load balancing. High availability decreases the chance of a loss of data when a single server fails. When a server fails, another server fills the gap by providing a copy of the data that was on the failed server. When the failed server comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster to improve the overall performance.
+You can also cluster both development and runtime environments. A clustered development or runtime environment consists of a unified group or "cluster" of two or more servers. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration, while the primary benefit of clustering {PRODUCT} runtime environments is high availability and load balancing. High availability decreases the chance of a loss of data when a single server fails. When a server fails, another server fills the gap by providing a copy of the data that was on the failed server. When the failed server comes online again, it resumes its place in the cluster.
```

Comment:
Again, I would change the instance of "a loss of data" to "data loss". Up to you.

Reply:
Done